### PR TITLE
Add DbValue for more flexible object parameters.

### LINKED ISSUE #13: Add DbValue class with JDBC Object Type and value
This will allow users to set a parameter to a query with a specific type to be set on setObject. This is necessary for certain use cases, such as custom types with enums in Postgres

### PR DESCRIPTION
**Pull request by chadmoller**

### CODE REVIEW

**Comment by chadmoller** [comment]
Closes #13

**Comment by chadmoller** [comment]
This will close #13

File: src/main/kotlin/com/target/liteforjdbc/PreparedStatementSetParameters.kt [hunk 1]
```diff
@@ -32,6 +29,64 @@ fun PreparedStatement.setInstant(index: Int, instant: Instant) {
...
+
+/**
+ * Sets OffsetTime parameter
+ */
+fun PreparedStatement.setOffsetTime(index:Int, value: OffsetTime) {
+    setObject(index, value)
+}
+
+/**
+ * Sets instant by converting it to LocalDateTime, assuming the timezone is UTC.
```
Comment (dtanner) [inline src/main/kotlin/com/target/liteforjdbc/PreparedStatementSetParameters.kt:68]: wrong comment?

**Review by dtanner — DISMISSED** [review]

File: src/main/kotlin/com/target/liteforjdbc/PreparedStatementSetParameters.kt (hunk 1 above)
Comment (chadmoller) [reply src/main/kotlin/com/target/liteforjdbc/PreparedStatementSetParameters.kt]: @dtanner yep, and resolved. The change invalidated your approval

**Review by chad-moller-target — APPROVED** [review]
Looks good

**Pull request merged**